src/Fixer/Phpdoc/PhpdocParamOrderFixer.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Phpdoc;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\DocBlock\Annotation;
+use PhpCsFixer\DocBlock\DocBlock;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Preg;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Jonathan Gruber <gruberjonathan@gmail.com>
+ */
+final class PhpdocParamOrderFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isTokenKindFound(T_DOC_COMMENT);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Orders all `@param` annotations in DocBlocks according to method signature.',
+            [
+                new CodeSample(
+                    '<?php
+class C {
+    /**
+     * @param bool   $a
+     * @param string $c
+     * @param string $b
+     */
+    public function m($a, $b, $c) {}
+}
+'
+                ),
+            ]
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function applyFix(\SplFileInfo $file, Tokens $tokens)
+    {
+        foreach ($tokens as $index => $token) {
+            if (!$token->isGivenKind(T_DOC_COMMENT)) {
+                continue;
+            }
+
+            // Check for function signature
+            $functionSequence = $tokens->findSequence([[T_FUNCTION], [T_STRING], '('], $index);
+
+            if (null === $functionSequence) {
+                continue;
```

Comment:
Since no `end` has been given for the sequence find the result `null` means no more functions are found in the token collection. Therefore I think a `break` or even a `return` can be done here.

Reply:
Alright, I used a `return`.